pkg/basicstation/cups/update_info.go

```diff
@@ -58,90 +60,86 @@ var (
 	errInvalidToken    = errors.DefinePermissionDenied("invalid_token", "invalid provisioning token")
 )
 
-var getGatewayMask = types.FieldMask{Paths: []string{
-	"attributes",
-	"version_ids",
-	"gateway_server_address",
-	"auto_update",
-	"update_channel",
-	"frequency_plan_id",
-}}
+func getAuthHeader(ctx context.Context) string {
+	if md, ok := metadata.FromIncomingContext(ctx); ok {
+		if authorization := md.Get("authorization"); len(authorization) > 0 {
```

Comment:
`Authorization` right?

Reply:
In gRPC (and http/2) headers are lowercase